pkg/build/builder/util.go

```diff
@@ -59,3 +59,29 @@ func getDockerNetworkMode() stiapi.DockerNetworkMode {
 	}
 	return ""
 }
+
+// MergeEnv will take an existing environment and merge it with a new set of
```

Comment:
@mfojtik I looked into merging this function with the one in the controller code. However, this one deals with vars as strings whereas the other one looks at kapi.EnvVars. It seemed that getting the code reuse would require more code to be written to convert from one to the other. If you still feel strongly about merging them I can try it, otherwise I'll leave it alone.